ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -27,7 +27,18 @@ export interface AccountValue {
   proofAuthorizingKey: string | null
 }
 
-export type AccountImport = Omit<AccountValue, 'id'>
+export type AccountImport = {
+  version: number
+  name: string
+  spendingKey: string | null
+  viewKey: string
+  incomingViewKey: string
+  outgoingViewKey: string
+  publicAddress: string
+  createdAt: HeadValue | null
+  multisigKeys?: MultisigKeysImport
+  proofAuthorizingKey: string | null
+}
 
```

Comment:
why this change? Seems like `Omit<AccountValue, 'id'>` would be cleaner than repeating the fields in two places?

Reply:
1. I don't really like `Omit` in this case because it's not clear that the import data could be slightly different.
2. The import data is slightly different. Multi sig keys can have either an identity or secret, but we store the secret.